### ISSUE TITLE
Test suite still fails with pyOpenSSL.dev0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8060]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#8060|
|Type|enhancement|
|Created|2015-10-21 12:16:16Z|
|Branch|https://github.com/twisted/twisted/tree/fix-RealOpenSSLTests-8060|

```
[FAIL]
Traceback (most recent call last):
  File "/Users/hynek/Projects/twisted/twisted/test/test_sslverify.py", line 2285, in test_oneOrTwoDots
    self.assertIn(self.OpenSSL.__version__.count("."), (1, 2))
  File "/Users/hynek/Projects/twisted/twisted/trial/_synctest.py", line 497, in assertIn
    % (containee, container))
twisted.trial.unittest.FailTest: 3 not in (1, 2)

twisted.test.test_sslverify.RealOpenSSLTests.test_oneOrTwoDots
```

<details><summary>Searchable metadata</summary>

```
trac-id__8060 8060
type__enhancement enhancement
reporter__hynek hynek
priority__normal normal
milestone__None None
branch__branches_fix_RealOpenSSLTests_8060 branches/fix-RealOpenSSLTests-8060
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1445429776978924 1445429776978924
changetime__1445435541797473 1445435541797473
version__None None
owner__hynek hynek

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_1"></a>|@hynek commented|
|-|-|

(In [45881]) Branching to fix-RealOpenSSLTests-8060.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

Bots are green, changes simple.  Grab your cheap review points!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hynek|
|-|-|

Changes looks good.

I think that the test_majorDotMinor is enought and we can remove the test.

With the current docstring I have no idea what is the purpose of this test.

Why should we care how many dots are in the pyOpenSSL version ?

So I think that is fair to remove the current test.

Please merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek set status to `closed`|
|-|-|

(In [45885]) Merge fix-RealOpenSSLTests-8060: that actually fixes OpenSSLVersionTestsMixin

Author: hynek
Reviewer: adiroiban
Fixes: [#8060](https://github.com/twisted/twisted/issues/8060)

Remove brittle test making too many assumptions about pyOpenSSL's versioning.
